open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/impl/DefaultMetaDataInfo.java

```diff
@@ -102,6 +104,11 @@
      */
     private boolean mySpecialKeyExaminationRequired = true;
 
+    /**
+     * The property in which the unique identifier key is maintained.
+     */
+    private final StringProperty myUniqueIdentifierKeyProperty = new SimpleStringProperty();

```

Comment:
If this property can be accessed from multiple threads, we should use our ConcurrentStringProperty here.

Reply:
Fixed.